modules/ROOT/pages/reconnection-strategy-about.adoc

```diff
@@ -14,13 +14,20 @@ You can modify this behavior by setting a number of reconnection attempts to try
 .Example with that fails after `5` reconnection attempts are exhausted:
 [source,xml,linenums]
 ----
-<http:request-config name="HTTP_Request_Config" doc:name="HTTP Request Config">
-  <http:request-connection host="https://jsonplaceholder.typicode.com/posts" port="80" >
+<ftp:config name="FTP_Config" doc:name="FTP Config" >
+  <ftp:connection host="${ftp.host}" port="${ftp.port}" username="${ftp.username}" password="${ftp.pwd}" >
```

Comment:
I would not use properties for these examples. IMO it will just cause confusion.

Reply:
just use constant values?